data_collection/gazette/spiders/mg_belo_horizonte.py

```diff
@@ -0,0 +1,54 @@
+import datetime
+import re
+
+import scrapy
+from dateutil.rrule import DAILY, rrule
+from urllib.parse import urlencode
+
+from gazette.items import Gazette
+from gazette.spiders.base import BaseGazetteSpider
+
+
+class MgBeloHorizonteSpider(BaseGazetteSpider):
+    TERRITORY_ID = "3106200"
+
+    name = "mg_belo_horizonte"
+    allowed_domains = ["dom-web.pbh.gov.br"]
+    start_date = datetime.datetime(2000, 1, 2)
+    end_date = None
+
+    def start_requests(self):
+        base_url = "https://api-dom.pbh.gov.br/api/v1/edicoes/buscarpublicacaopordata"
+
+        end_date = self.end_date or datetime.date.today()
+
+        rule_start_date = datetime.date(
+            self.start_date.year, self.start_date.month, self.start_date.day
+        )
+        date_list = list(rrule(freq=DAILY, dtstart=rule_start_date, until=end_date))
+        for date in date_list:
```

Comment:
I'm not sure about this cast to `list`. Do we need it?

Also, the variable was created to improve legibility, right? You think we should avoid using `rrule` on its own?

Reply:
No reason. Maybe I was debugging the result and created the variable. I removed it as it won't be needed.